### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by ledamint**

### CODE REVIEW

File: packages/urban-bot/src/types/Messages.ts [hunk 1]
```diff
@@ -168,7 +167,7 @@ export type UrbanMessageMedia = UrbanMessageCommon & {
 export type UrbanMessageLocationData = UrbanMessageCommonData & {
     latitude: number;
     longitude: number;
-    livePeriod?: number;
+    livePeriodSeconds?: number;
```
Comment (oTkPoBeHuE) [inline packages/urban-bot/src/types/Messages.ts:170]: `livePeriodSeconds ` or `livePeriodsSeconds`?

File: docs/DOCS.md [hunk 2]
```diff
@@ -0,0 +1,1018 @@
...
+```javascript
+import { render, Root, Text } from '@urban-bot/core';  
+```
+```javascript
+const { render, Root, Text } = require('@urban-bot/core');  
+```  
+## render  
+The main function that starts React. Works similar `ReactDOM.render`.  
+### Arguments  
+####  1
```
Comment (oTkPoBeHuE) [inline docs/DOCS.md]: It's not clear.

File: docs/DOCS.md [hunk 3]
```diff
@@ -0,0 +1,1018 @@
...
+    }  
+/>  
+```  
+#### parseMode
+>  The markup language which is used for parsing text. Calculated automatically for every messenger, but you can specify directly.
+###### optional  
+`'HTML'` | `'markdown'`
+```javascript
+<Text parseMode="HTML">
+    <b>bold</b> // '<b>bold</b>'
```
Comment (oTkPoBeHuE) [inline docs/DOCS.md]: it's not clear.

File: packages/urban-bot/src/types/Messages.ts (hunk 1 above)
Comment (ledamint) [reply packages/urban-bot/src/types/Messages.ts:170]: 1

File: docs/DOCS.md (hunk 3 above)
Comment (ledamint) [reply docs/DOCS.md]: fixed

**Pull request merged**